src/components/study/studySchedule/TodoList.tsx

```diff
@@ -1,19 +1,24 @@
 import { getStudyTask } from '@/api/study/studyAPI';
 import { useQuery } from '@tanstack/react-query';
-import type { FC } from 'react';
+import { Dispatch, FC, SetStateAction, useEffect } from 'react';
 
 interface Props {
   taskDate: string;
   studyId: number;
+  setTaskId: Dispatch<SetStateAction<number>>;
 }
 
-const TodoList: FC<Props> = ({ taskDate, studyId }: Props) => {
+const TodoList: FC<Props> = ({ taskDate, studyId, setTaskId }: Props) => {
   const { data } = useQuery({
     queryFn: () => getStudyTask(Number(studyId), taskDate),
     queryKey: ['studyTasks', taskDate],
     retry: 1,
   });
 
+  useEffect(() => {
+    if (data?.data) setTaskId(data?.data.taskId);
```

Comment:
```suggestion
    if (data?.data) setTaskId(data.data.taskId);
```
해당 if문 내에 들어가면 옵셔널 체이닝이 필요 없을 것 같네요

Reply:
아 ..!! 넵 감사합니다 !!